Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml

```diff
@@ -265,6 +265,9 @@
     sprite: Clothing/OuterClothing/Misc/hospitalgown.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Misc/hospitalgown.rsi
+  - type: Tag
+    tags:
+    - PermissibleForSurgery # DeltaV: tank harnesses can be used for surgery
```

Comment:
this bulldozes chameleon whitelist and maybe other tags, the comment should also go on the `type: Tag` line

Reply:
added chameleon whitelist, there's no other tags on the parents